packages/vm/src/testing/mock.rs

```diff
@@ -130,10 +132,13 @@ impl Api for MockApi {
             .filter(|&x| x != 0)
             .collect();
 
-        match String::from_utf8(trimmed) {
-            Ok(human) => Ok((HumanAddr(human), gas_info)),
-            Err(err) => Err((err.into(), gas_info)),
-        }
+        (
+            match String::from_utf8(trimmed) {
+                Ok(human) => Ok(HumanAddr(human)),
+                Err(err) => Err(err.into()),
+            },
+            gas_info,
+        )
```

Comment:
I'd assign the match statement to a variable and then build the tuple.

Reply:
Done in https://github.com/CosmWasm/cosmwasm/pull/489/commits/5e726fa615d7f33fa28dad4aae0bf4cbd2f27516